Toolchain/flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  description = "Serenity OS";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+  outputs = { self, flake-utils, nixpkgs }:
```

Comment:
Does this flake file work just as well for Ladybird/ladybird.nix? I wonder if there's a way to use the same info for both of those environments. one for building the OS and running in QEMU, another for building the browser on your host and running it with Qt6

Reply:
If it works for both, it might make sense to drop it into `Meta/` instead? to centralize.

Reply:
This was made to build the OS, however there is nothing stopping us from adding another dev shell here specifically for building Ladybird. Should be easy to do. If you want to see an example of a shell with multiple shells have a look (towards the end) at https://github.com/ocaml/dune/blob/main/flake.nix

Reply:
Sure, room for improvement

Reply:
I've opened https://github.com/SerenityOS/serenity/pull/20890 for adding the ladybird shell.